### PR TITLE
Unsuscribe

### PR DESCRIPTION
**Pull request by EduBeghe**
- Added ui for unsuscribe 
- Added queries and logic for unsuscribe functionality
- Added unsuscribeEmails column for business table
- Fixed login bug

### CODE REVIEW

File: backend/queries.ts [hunk 1]
```diff
@@ -478,4 +478,14 @@ export const getFederalDueDates = async function (): Promise<any> {
     } catch(error) {
         console.log(error);
     }
+}
+
+export const unsuscribe = async function (email: string): Promise<any> {
```
Comment (MichaelAshmead) [inline backend/queries.ts:483]: Fix typo: Should be "unsubscribe"

Comment (MichaelAshmead) [reply backend/queries.ts:483]: See this typo in a few spots, so just make sure you make a global update

Comment (EduBeghe) [reply backend/queries.ts]: Fixed

File: db_migrations/v014_unsuscribe_column.sql [hunk 2]
```diff
@@ -0,0 +1,2 @@
+ALTER TABLE `Production`.`business` 
+ADD COLUMN `unsuscribeEmails` TINYINT NOT NULL DEFAULT 0 AFTER `plaidLastPull`;
```
Comment (MichaelAshmead) [inline db_migrations/v014_unsuscribe_column.sql:2]: One more typo :)

**Review by MichaelAshmead — APPROVED** [review]

**Pull request merged**